src/exercise/client/src/Components/BottomButtons/BottomButtons.js

```diff
@@ -28,11 +33,11 @@ export default function BottomButtons({
     const tmp = [...tasks];
     tmp.sort((a, b) => {
       if (ascAZ) {
-        return a.task.localeCompare(b.task);
+        return a.ItemName.localeCompare(b.ItemName);
```

Comment:
the sorting logic should be moved to the reducer - by doing so you make sure that no matter where the info gets into the redux state, it will be sorted